department-of-veterans-affairs/next-build
# Fix sidenavs not redering for lovell facility pages

## Description
Relates to #[14745](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/14745). (or closes?)


## Testing done
Local visual testing

## Screenshots
<img width="1213" alt="Screen Shot 2023-10-31 at 1 54 59 PM" src="https://github.com/department-of-veterans-affairs/next-build/assets/61624970/c03b984c-930d-41da-9501-dcd320992fe6">
<img width="1213" alt="Screen Shot 2023-10-31 at 1 55 27 PM" src="https://github.com/department-of-veterans-affairs/next-build/assets/61624970/5646ebde-47e4-4729-8243-82ae19d7439a">


## QA steps
Navigate to lovell pages (/lovell-federal-health-care-tricare/stories/)
Verify that sidenav is rendering
Verify that links are correct with no duplicates
Verify that non-lovell page sidenavs are unchanged (/butler-health-care/stories/)


## Is this PR blocked by another PR?
- Add the `DO NOT MERGE` label
- Add links to additional PRs here:
